### PR TITLE
GSoD 2020

### PR DESCRIPTION
**Pull request by satu0king**
GSoD 2020 changes

### CODE REVIEW

File: assets/css/just-the-docs-circuitversedark.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+---
+{% include css/just-the-docs.scss.liquid color_scheme="circuitversedark" %}
```
[the remark this answers is not in the document]
Comment (manjotsidhu) [reply assets/css/just-the-docs-circuitversedark.scss:3]: Ignore, https://pmarsceill.github.io/just-the-docs/docs/customization/#switchable-custom-scheme

File: assets/css/just-the-docs-circuitverse.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+---
+{% include css/just-the-docs.scss.liquid color_scheme="circuitverse" %}
```
[the remark this answers is not in the document]
Comment (manjotsidhu) [reply assets/css/just-the-docs-circuitverse.scss:3]: Ignore, https://pmarsceill.github.io/just-the-docs/docs/customization/#switchable-custom-scheme

**Review by Shivansh2407 — APPROVED** [review]
Looks good to me :+1: . @satu0king @tachyons @nitin10s Please have a look at it so that we can merge this :tada: .
Thank you @danielluehr for participating in GSoD 2020 under Circuitverse and completely revamping the Interactive Book. Also a big thanks to @manjotsidhu @ayan-biswas0412 @anushri20  @infern018 @Samridhi-98 @ritik307 @Rish1234567 for fixing bugs and enhancing this project!

**Comment by manjotsidhu** [comment]
After merge, please immediately change `url` in `config.yml`. We would also need to switch to GitHub Pages from Netlify.

**Comment by Shivansh2407** [comment]
> After merge, please immediately change `url` in `config.yml`. We would also need to switch to GitHub Pages from Netlify.

Yes! 
cc: @satu0king

File: CNAME [hunk 3]
```diff
@@ -1 +1 @@
-learn.circuitverse.org
\ No newline at end of file
+betalearn.circuitverse.org
```
Comment (satu0king) [inline CNAME:1]: Change this

Comment (manjotsidhu) [reply CNAME]: Done

**Comment by manjotsidhu** [comment]
@satu0king done

**Pull request merged**